client/src/components/ProjectSamplesTable.js

```diff
@@ -86,16 +83,11 @@ export const ProjectSamplesTable = ({
           justify="around"
           width={checkBoxCellWidth}
         >
-          {allModalities.map((m) => (
-            <ModalityCheckBox
+          {availableModalities.map((m) => (
```

Comment:
Can we move this mapping into the `ProjectSamplesTableModalityCell` component. I think you can also move some of the layout styles from above into the child component.